authentication/src/main/AndroidManifest.xml

```diff
@@ -1,18 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.udinic.accounts_authenticator_example"
+          package="com.udinic.accounts"
```

Comment:
oh wait, we cannot change the package name at all :)
http://stackoverflow.com/questions/17582289/is-it-possible-to-change-the-package-name-of-an-android-app-on-google-play